Energy Efficiency EACs Using Pre and Post Measurements.md

```diff
@@ -0,0 +1,516 @@
+n* = \frac {Nn} {N+n}
+```
+    where; 	
+    n* = adjusted sample size (actual minimum sample fixtures) 
+    n = initial sample size (calculated above) 
+    N = population size (total fixtures)
+
+Resulting in the following minimum sample sizes based on total fixtures to achieve a 10% precision at 90% confidence:
+
+
+<table>
+  <tr>
+   <td><strong>Total Fixture Quantity Retrofit (N)</strong>
+   </td>
+   <td><strong>Minimum Sample Fixture Quantity (n*)</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>3
+   </td>
+   <td>                         3 
+   </td>
+  </tr>
+  <tr>
+   <td>5
+   </td>
+   <td>                         5 
+   </td>
+  </tr>
+  <tr>
+   <td>10
+   </td>
+   <td>                         9 
+   </td>
+  </tr>
+  <tr>
+   <td>25
+   </td>
+   <td>                       19 
+   </td>
+  </tr>
+  <tr>
+   <td>50
+   </td>
+   <td>                       29 
+   </td>
+  </tr>
+  <tr>
+   <td>100
+   </td>
+   <td>                       41 
+   </td>
+  </tr>
+  <tr>
+   <td>250
+   </td>
+   <td>                       54 
+   </td>
+  </tr>
+  <tr>
+   <td>500
+   </td>
+   <td>                       60 
+   </td>
+  </tr>
+  <tr>
+   <td>1000
+   </td>
+   <td>                       64 
+   </td>
+  </tr>
+  <tr>
+   <td>2500
+   </td>
+   <td>                       66 
+   </td>
+  </tr>
+  <tr>
+   <td>5000
+   </td>
+   <td>                       67 
+   </td>
+  </tr>
+  <tr>
+   <td>≥7500
+   </td>
+   <td>                       68 
+   </td>
+  </tr>
+</table>
+
+
+Where baseline data logging is conducted, the same circuits will be monitored post retrofit for side-by-side comparison.  In such cases, the difference between measured post energy consumption to baseline energy consumption will be used to verify the true reduction;
+
+
+    _Saved Annual Electricity (kWh/yr) = Measured Baseline Annual Electricity – Measured Proposed Annual Electricity_
+
+Where 
+
+
+    _Measured Baseline Annual Electricity = ∑ Measured Baseline Circuit kW x Measurement Interval (minutes) / 60 minutes per hour **and**_
+
+
+    _Measured Proposed Annual Electricity = ∑ Measured Proposed Circuit kW x Measurement Interval (minutes) / 60 minutes per hour <span style="text-decoration:underline;">if power</span>_ is measured
+
+_Or_
+
+
+    _Measured Baseline Annual Electricity = ∑ Measured Baseline Circuit Amperage x Baseline Circuit Voltage / 1000 W per kW x Measurement Interval (minutes) / 60 minutes per hour **and**_
+
+
+    _Measured Proposed Annual Electricity = ∑ Measured Baseline Circuit Amperage x Baseline Circuit Voltage / 1000 W per kW x Measurement Interval (minutes) / 60 minutes per hour <span style="text-decoration:underline;">if current</span>_ is measured
+
+Measurements will be taken for a full year during the Proposed period, and optimally for a full year in the Baseline period.  However, if the baseline monitoring period is not one full year, the Baseline Period Electricity will be extrapolated to annual consumption;
```

Comment:
Would recommend including either a minimum time or how the extrapolation might work.